recipes/lora_finetune_single_device.py

```diff
@@ -42,13 +44,21 @@ class LoRAFinetuneRecipeSingleDevice(FTRecipeInterface):
     for single GPU training. Training on CPU is not supported.
 
     Features:
-        - Activation Checkpointing. This can be controlled using the ``activation_checkpointing``
+        - Activation Checkpointing. This can be controlled using the ``enable_activation_checkpointing``
             flag. Activation checkpointing helps reduce the memory footprint since we no longer keep
             activations in memory and instead recompute them during the backward pass. This is especially
             helpful for larger batch sizes when you're memory constrained. But these savings in memory
             come at the cost of training performance. In most cases training can slow-down quite a bit as
             a result of this activation recomputation.
 
+        - Activation Offloading. This can be controlled using the ``enable_activation_offloading``
+            flag. Activation offloading helps reduce the memory footprint since we no longer keep
+            activations in memory and instead offload them to CPU and bring them back during the
+            backward pass. This is especially helpful for larger batch sizes when you're memory
+            constrained. But these savings in memory come at the cost of training performance and
+            CPU resources. In most cases, activation offloading is most effective when used in
+            conjunction with activation checkpointing.
```

Comment:
> "In most cases, activation offloading is most effective when used in conjunction with activation checkpointing." 

nit: can we add 1 sentence here explaining why? I think it may be worth to add to the very beginning that it can be used independently of checkpointing.